assets/styl/_modules/buttons-icon-socials.json

```diff
@@ -0,0 +1,18 @@
+{
+    "socials" : {
+        "messenger" : "Messenger",
+        "facebook" : "Facebook",
+        "twitter" : "Twitter",
+        "linkedin" : "LinkedIn",
+        "googleplus" : "Google Plus",
+        "pinterest" : "Pinterest",
+        "whatsapp" : "What's App",
+        "sms" : "SMS",
+        "hootsuite" : "Hootsuite",
+        "pocket" : "Pocket",
+        "github" : "Github",
+        "snapchat" : "Snapchat",
+        "whatsapp" : "Whatsapp"
+
+    }
+}
```

Comment:
I already tried this. But it's not good since when we define modifiers, KSS always create a default view, and in this case, I don't want a default view because the pictos are modifiers too. The default view would be without a picto, and it leaves a blank space. It gives an example that will never be used. And I didn't find a way to hide or suppress the default first element so I did it like that.